submissions/balancer-design/universal_8to8.md

```diff
@@ -0,0 +1,68 @@
+## Universal 8-8: Perfectly balanced, as all things should be
+
+Some time ago I made a bunch of computer parts with belts and splitters. This time I will continue my unhealthy obsession with moving items around by making something that's actually at least slightly practical, that being a balancer. Now, why would I write a whole article on designing a balancer? Well that is because the one we will be making today is no average balancer. This article won't go very deep into balancer design principles, since we will be using pre-made and well-known blueprints for the most part. What you *do* need to know is some terminology:
+
+* Input-balanced: Draws from all inputs evenly. Especially important when merging many inputs to a few outputs.
+* Output-balanced: Gives to all outputs evenly. Especially important when splitting a few inputs to many outputs.
+* Throughput-limited: Has an input-output configuration that creates an internal bottleneck. For example, merging two belts into one and splitting back to two.
+* Throughput-unlimited: Opposite of throughput-limited.
+
+All blueprints from this article (and a couple extras) can be found at [this link](https://controlc.com/c21ae405).
+
+### The premise
+
+Picture the following: you're a new player, and you've just heard about those things called "balancers" and how useful they are. You see an image of a 4-4 balancer, build it, but then only hook up 3 outputs to get 3 balanced belts. Instead you're met with a 2:1:1 ratio and disappointment:
+
+![How not to use a balancer](https://cdn.discordapp.com/attachments/699618487097884723/813449477028446269/unknown.png)
+
+In this article, I set myself a goal to make a balancer for which this sort of intuition works, in other words, no matter the input/output configuration, all outputs are always equal, and all inputs are drawn from evenly. This kind of balancer is known as a "universal balancer", a concept first explored and named in [this post](https://www.reddit.com/r/factorio/comments/a5ferf/i_present_to_you_the_44_universal_balancer/) by Reddit user [u/tzwaan](https://www.reddit.com/user/tzwaan), who is one of the moderators of the [Factorio subreddit](https://www.reddit.com/r/factorio).
+
+Universal balancers can do what no balancer should be able to: no matter the configuration, they're input-balanced, output-balanced, *and* throughput-unlimited at the same time! This is a set of properties previously thought to be possessed only by a 2-2 balancer, which is just a single splitter. The drawback is that they get *big*.
+
+### The design process
+
+Let's think about how to convert a 1-4 balancer into a 1-3. That is done by looping the extra output back into the first splitter. The same can be done with a 4-4 balancer, which we will use as our base principle:
```

Comment:
I think it should be made clear that you're not building 1-3 balancer. But rather showing a looped-back 3-3 balancer as an example, and then - as you said - use it as a base principle. These ideas could be confusing, so it's important we hold the reader's hand through the explanation :P